client/users/profile/profile.ng.jade

```diff
@@ -0,0 +1,52 @@
+.profile
+    .form-container(layout="row", layout-sm="column", layout-align="center center")
+        md-whiteframe.md-whiteframe-z1.username-form
+            ng-form(name="prof.loginForm" layout="column")
+                md-input-container
+                    input(type="text" ng-model="currentUser.username" name="username"
+                            placeholder="Login" aria-label="login")
+                md-button.md-raised.md-primary(ng-click="prof.actions.updateUsername(currentUser.username)"
+                        aria-label="update login") Update login
+        md-whiteframe.md-whiteframe-z1.email-form
+            ng-form(name="prof.emailForm" layout="column")
+                md-input-container
+                    input(type="text" ng-model="currentUser.emails[0].address" name="email"
+                            placeholder="E-mail" aria-label="email")
+                md-button.md-raised.md-primary(ng-click="prof.actions.updateEmail(currentUser.emails[0].address)"
+                        aria-label="update email") Update email
+    .form-container(layout="row", layout-sm="column", layout-align="center center")
+        md-whiteframe.md-whiteframe-z1
+            ng-form(name="prof.emailForm")
+                h4.title Profile info
+                .names(layout="row", layout-sm="column", layout-align="left center")
+                    md-input-container
+                        input(type="text" ng-model="currentUser.profile.firstname" name="firstname"
+                                placeholder="Firstname" aria-label="first name")
+                    md-input-container
+                        input(type="text" ng-model="currentUser.profile.lastname" name="lastname"
+                                placeholder="Lastname" aria-label="last name")
+                h4.title Do you have a cat?
```

Comment:
No przecież od początku miało być o kotach. Dodałem coś, żeby to jakoś wyglądało ładniej :smile_cat: